### PR TITLE
Update to use GoFSH 1.3.0

### PR DESCRIPTION
**Pull request by ngfreiter**
This updates FSHOnline to use GoFSH 1.3.0. Some of the functions changed slightly and needed to be updated, and the boxes at the end of the output also needed to be edited to match what GoFSH now logs.

### CODE REVIEW

File: src/utils/FSHHelpers.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ const stats = utils.stats;
 const getRandomPun = utils.getRandomPun;
 const Type = utils.Type;
 const FHIRDefinitions = fhirdefs.FHIRDefinitions;
-const { getRandomSeaCreatures, getRandomSeaCreaturesStat } = gofshUtils;
```
Comment (jafeltra) [inline src/utils/FSHHelpers.js:15]: Bye sea creatures. 👋

**Review by jafeltra — APPROVED** [review]
Looks good to me! It's nice to have the round trip back!

**Comment by jafeltra** [comment]
Gah wait I thought of something else. Since we now support profiles of logical models, we might want to update the sorting we do as a part of this PR.

**Comment by ngfreiter** [comment]
> Gah wait I thought of something else. Since we now support profiles of logical models, we might want to update the sorting we do as a part of this PR.
> 
> **Update**: I actually think this should be very easy, so I vote we add it into this PR. The function `checkFshType` [here](https://github.com/FSHSchool/FSHOnline/blob/master/src/components/JSONOutput.js#L81) just needs to be udpated so that we include `'logical'` as an allowed `kind` for profiles. So I think we can just update [line 100](https://github.com/FSHSchool/FSHOnline/blob/master/src/components/JSONOutput.js#L100) to be
> 
> ```
> } else if (['resource', 'complex-type', 'primitive-type', 'logical'].includes(def.kind)) {
> ```

Good catch! Done in https://github.com/FSHSchool/FSHOnline/pull/68/commits/b8582a50bf105e3545ea0601014e364622903f5d.

**Review by jafeltra — APPROVED** [review]
This looks good! Thanks for making that change!

**Review by julianxcarter — APPROVED** [review]

**Pull request merged**